saritasa_invocations/git.py

```diff
@@ -48,3 +50,248 @@ def clone_repo(
         printing.print_success(f"Pulling changes for {repo_link}...")
         with context.cd(repo_path):
             context.run("git pull")
+
+
+@invoke.task
+def blame_copy(
+    context: invoke.Context,
+    original_path: str,
+    destination_paths: str,
+) -> None:
+    """Copy file from original path to destination paths with saving blame.
+
+    If destination path is file,
+    then data will be copied in it.
+    If destination path is directory,
+    then data will be copied in provided directory with original name.
+    How script works:
+        1) Remember current HEAD state
+        2) For each copy path:
+            move file to copy path, restore file using `checkout`,
+            remember result commits
+        3) Restore state of branch
+        4) Move file to temp file
+        5) Merge copy commits to branch
+        6) Move file to it's original path from temp file
+    Count of created commits:
+        N + 3, where N - is count of copies,
+        3 - 1 commit to put original file to temp file
+        1 commit to merge commits with creation of copy files
+        1 commit to put data from temp file to original file back.
+
+    """
+    config = _config.Config.from_context(context)
+    destination_paths_list = _split_destination_paths(destination_paths)
+    _validate_paths(original_path, destination_paths_list)
+    printing.print_success(
+        config.git.copy_init_message_template.format(
+            original_path=original_path,
+            destination_paths="\n* ".join(destination_paths_list),
+            commits_count=len(destination_paths_list) + 3,
+        ),
+    )
+
+    input("Press any key to continue\n")
```

Comment:
Why it's needed?

Reply:
Due command creates files and commits during its execution, I thought it would be useful to let user know which file he choose and count of commits that would be created after execution. In case if user made some mistake, he can just `ctrl+c` instead of write `git reset --hard HEAD~N` after command execution.

Reply:
Click which comes with invoke has confirm shortcut
https://click.palletsprojects.com/en/8.1.x/prompts/

Reply:
due click not provided by invoke, we will not use it here (as discussed in slack)